source/transactions/tests/transaction-options-repl.yml

```diff
@@ -0,0 +1,117 @@
+runOn:
+  - minServerVersion: "4.0"
+    topology: ["replicaset"]
+
+database_name: &database_name "transaction-tests"
+collection_name: &collection_name "test"
+
+data: []
+
+tests:
+
+  - description: readConcern snapshot in startTransaction options
```

Comment:
This is the only test that _needs_ to send 'snapshot' to the server so I've left it unchanged and simply skipped it on sharded clusters (`runOn` requires this test to only run on a replica set).